overrides/main.html

```diff
@@ -0,0 +1,8 @@
+{% extends "base.html" %}
+
+{% block outdated %}
```

Comment:
Will this show on latest release i.e 0.6.1?

Reply:
Ha, was still a draft - I've turned the other way around (See latest screenshots). 0.7.0 now has an unstable warning (0.6.1 will be marked as latest/stable)

Reply:
Awesome, thanks!